app/controllers/api/v1/events/nearests_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::Events::NearestsController < ApiController
+  def index
+    @events = Event.near(
+      [params[:lat], params[:lon]],
```

Comment:
Might be nice and slight more readable to have a private method called `coordinates` that returns the array `[params[:lat], params[:lon]]` so that can use:

`Event.near(coordinates, params[:radius ...)`